### PR TITLE
feat(yandex-metrika): update loading script

### PR DESCRIPTION
**Pull request by x-range**
I'm fixed loading code of yandex metrika. 
Error at issue [#364](https://github.com/nuxt-community/modules/issues/364) and [#362](https://github.com/nuxt-community/modules/issues/362)

### CODE REVIEW

**Comment by pi0** [comment]
Hi @x-range would you please rebase your changes with master branch? #363 is already merged....

**Comment by x-range** [comment]
Hi @pi0 i rebased master brunch now.

File: packages/yandex-metrika/plugin.js [hunk 1]
```diff
@@ -7,11 +7,15 @@ export default ({ app: { router } }) => {
   })
 
   function create() {
-    ym(<%= options.id %>, "init", <%= JSON.stringify(options) %>);
+
     router.afterEach((to, from) => {
       if (!ready) {
         // Don't record a duplicate hit for the initial navigation.
-        return
+        (function(m,e,t,r,i,k,a){m[i]=m[i]||function(){(m[i].a=m[i].a||[]).push(arguments)};
```
Comment (pi0) [inline packages/yandex-metrika/plugin.js:14]: I think this is a wrong place to init script (notice `router.afterEach`)

Comment (x-range) [reply packages/yandex-metrika/plugin.js]: You're right. I'm fixed it.

**Pull request merged**